install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -107,23 +107,28 @@ data:
     # traffic. Set this field to tweak the period that Envoy will wait
     # for the client to send the first bits of data. (MUST BE >=1ms)
     protocolDetectionTimeout: {{ .Values.global.proxy.protocolDetectionTimeout }}
-    
+
     # DNS refresh rate for Envoy clusters of type STRICT_DNS
     dnsRefreshRate: {{ .Values.global.proxy.dnsRefreshRate }}
 
     # Unix Domain Socket through which envoy communicates with NodeAgent SDS to get
-    # key/cert for mTLS. Use secret-mount files instead of SDS if set to empty. 
+    # key/cert for mTLS. Use secret-mount files instead of SDS if set to empty.
     sdsUdsPath: {{ .Values.global.sds.udsPath }}
 
     # The trust domain corresponds to the trust root of a system.
     # Refer to https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md#21-trust-domain
     trustDomain: {{ .Values.global.trustDomain }}
 
+    # If set to true, and a given service does not have a corresponding DestinationRule configured,
+    # or its DestinationRule does not have TLSSettings specified, Istio configures client side
+    # TLS configuration appropriately.
```

Comment:
mention that by default it is true

Reply:
Done

Reply:
Revert. We want default to `false` before all implementation complete and well tested.

Reply:
I think we can recommend or shed some light on this will be planed as default behavior in future releases. So it motivate customers better to opt-in earlier, and have a better preparation for the migration.

Just a thought.

Reply:
IMO, it's better to do that when we ready to make it default. But if you have strong opinion about it, I can add it.